pkgs/development/python-modules/btrfs/default.nix

```diff
@@ -1,22 +1,31 @@
 { lib
 , buildPythonPackage
-, fetchPypi
+, fetchFromGitHub
 }:
 
 buildPythonPackage rec {
   pname = "btrfs";
-  version = "11";
+  version = "12";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "1w92sj47wy53ygz725xr613k32pk5khi0g9lrpp6img871241hrx";
+  src = fetchFromGitHub {
+    owner = "knorrie";
+    repo = "python-btrfs";
+    rev = "v${version}";
+    sha256 = "sha256-ZQSp+pbHABgBTrCwC2YsUUXAf/StP4ny7MEhBgCRqgE=";
   };
 
+  postInstall = ''
+    mkdir -p "$out/bin"
+    mv bin/btrfs-* "$out/bin"
```

Comment:
huh, i missed these, doesn't look like the v11 pypi package had these though

Reply:
Can you ask the upstream about setting up entrypoints/scripts?

https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html

Reply:
https://github.com/knorrie/python-btrfs/pull/30

Please fetchpatch and we drop this after the next release.

Reply:
I'll wait until your PR is merged, then I'll update this PR.

Reply:
It is already merged and will be part of v13.

Reply:
Thank you for your suggestion.